spec/index.html

```diff
@@ -592,11 +592,11 @@ <h2>Literals</h2>
       Simple literals are syntactic sugar for abstract syntax
       <a>literals</a>
       with the <a>datatype IRI</a>
-      <code>http://www.w3.org/2001/XMLSchema#string</code>. Similarly, most
-      concrete syntaxes represent
+      <code>http://www.w3.org/2001/XMLSchema#string</code> (<code>xsd:string</code>).
```

Comment:
Change along the lines of @TallTed's suggestion in https://github.com/w3c/rdf-concepts/issues/22#issuecomment-1494927596

```suggestion
      <code>http://www.w3.org/2001/XMLSchema#string</code>
      (which is commonly abbreviated as <code>xsd:string</code>).
```